packages/nuxt/src/core/builder.ts

```diff
@@ -61,7 +61,7 @@ function watch (nuxt: Nuxt) {
 async function bundle (nuxt: Nuxt) {
   try {
     const { bundle } = typeof nuxt.options.builder === 'string'
-      ? await importModule(nuxt.options.builder, { paths: nuxt.options.rootDir })
+      ? await importModule(nuxt.options.builder, { paths: [import.meta.url, nuxt.options.rootDir, nuxt.options.workspaceDir] })
```

Comment:
I would put rootDir before so that we pick user version but has same effect i guess with webpack

Reply:
```suggestion
      ? await importModule(nuxt.options.builder, { paths: [nuxt.options.rootDir, nuxt.options.workspaceDir, import.meta.url] })
```